Source/ULibs.TinyJsonDeser/JsonDeserializer.cs

```diff
@@ -161,7 +161,7 @@ public bool TryParseArray(string json, out object[] output)
         /// <param name="json">The json string to parse.</param>
         /// <param name="output">If the parse succeeded, this will hold the parsed result.</param>
         /// <returns>Whether or not the parse attempt succeeded.</returns>
-        public bool TryParseObject(string json, out IDictionary<string, object> output)
+        public bool TryParseObject(string json, out IDictionary<string, object?>? output)

```

Comment:
Same. You're saying the object in the dictionary can't be `null`, but it can, and deliberately so. No wonder you're struggling with this PR!